### PR TITLE
Feature - Add article readtime

### PR DESCRIPTION
**Pull request by JLCarveth**
As per Issue #59, displays the calculated article read time. Can be disabled by using the `disableReadtime` option.

### CODE REVIEW

**Comment by JLCarveth** [comment]
This is my first real PR to an OS repo, so please let me know if I did something horribly wrong 😅

**Review by bartlomieju — COMMENTED** [review]
Cool! Can you update one of the tests that checks that "min read" is present in the response?

**Comment by JLCarveth** [comment]
3 of the tests are failing, specifically when it's looking for `<time ...` within the post body. However I tested and these were failing before my changes as well. I could likely fix that but it would be a new PR.

File: blog.tsx [hunk 1]
```diff
@@ -266,6 +266,7 @@ async function loadPost(postsDirectory: string, path: string) {
     ogImage: data.get("og:image"),
     tags: data.get("tags"),
     allowIframes: data.get("allow_iframes"),
+    readTime: readingTime(content),
   };
   POSTS.set(pathname, post);
   console.log("Load: ", post.pathname);
```
Comment (ije) [inline blog.tsx:272]: let's remove the debug message

File: types.d.ts [hunk 2]
```diff
@@ -76,6 +76,8 @@ export interface BlogSettings {
   port?: number;
   /** The hostname to serve the blog on */
   hostname?: string;
+  /** When true, readtime will not be displayed */
+  disableReadtime?: boolean;
```
Comment (ije) [inline types.d.ts:80]: i prefer to use `showReadtime` (false by default) instead of `disableReadtime`

Comment (roj1512) [reply types.d.ts:80]: I prefer to make it even simpler:

```ts
/** Whether to display read time or not */
readTime?: boolean;
```

Comment (JLCarveth) [reply types.d.ts]: Changed `disableReadtime` to `readtime`. Default is false / no readtime shown.

**Review by ije — APPROVED** [review]
LGTM! thanks!

**Comment by roj1512** [comment]
@JLCarveth Can you run

```shell
deno test
```

and investigate what's wrong with it?

**Comment by roj1512** [comment]
@ije @bartlomieju Approve the workflow to be run so we can see if there's any problem.

**Pull request merged**